external/llvm-project/mlir/include/mlir/Dialect/Tosa/Transforms/Passes.h

```diff
@@ -42,29 +42,8 @@ std::unique_ptr<Pass> createTosaMakeBroadcastablePass();
 std::unique_ptr<Pass> createTosaTestQuantUtilAPIPass();
 std::unique_ptr<Pass> createTosaOptionalDecompositions();
 std::unique_ptr<Pass> createTosaPartitionPass();
-
-class TosaPartitionPass : public TosaPartitionBase<TosaPartitionPass> {
```

Comment:
@pcf000 should this remain for other users? Does it need a test.

Reply:
I've replaced all the tests with appropriate strings of options